starters/nextjs-starter-ts/components/smart-components/index.ts

```diff
@@ -10,6 +10,7 @@ export const serverSmartComponentMap = {
         displayName: "Title",
         required: true,
         type: "string",
+        defaultValue: "Some title",
```

Comment:
Why do we need default value for Title field?

Reply:
It was accidentally included in the changes. Good catch.